chore: fix typos (#1012)
rupali-codes/LinksHub: CONTRIBUTING.md

- [Node.js](https://nodejs.org/) is installed.

---

## How to Contribute 🤔

- Look at the existing [**Issues**](https://github.com/rupali-codes/LinksHub/issues) or [**create a new issue**](https://github.com/rupali-codes/LinksHub/issues/new/choose)!

> - [**Fork the Repo**](https://github.com/rupali-codes/LinksHub/fork). Then, create a branch for any issue that you are working on. Finally, commit to your work.

Reviewer: [Proofreading]
```suggestion
- [**Fork the Repo**](https://github.com/rupali-codes/LinksHub/fork). Then, create a branch for any issue that you are working on. Finally, start working! 😊
```
